rb/build.desc

```diff
@@ -158,6 +158,15 @@ ruby_test(name = "ie",
   deps = [ ":ie" ]
 )
 
+ruby_test(name = "ie-w3c",
+  srcs = [
+    "spec/integration/selenium/webdriver/*_spec.rb",
+    "spec/integration/selenium/webdriver/ie/**/*_spec.rb"
+  ],
+  include = ["rb/spec/integration", "build/rb/lib"],
+  deps = [ ":ie" ]
+)
+
```

Comment:
Since this is using the w3c bridge, it will need:

    
    resources = [
        { "//javascript/webdriver/atoms:getAttribute": "rb/lib/selenium/webdriver/atoms/getAttribute.js"}
      ],

Added to https://github.com/SeleniumHQ/selenium/blob/e5e00222491d0439c8b5a64ad3ed8a35d344282f/rb/build.desc#L144

At some point, should we just put that into `:common` as it would be needed for all w3c compliant drivers?